test/libsolidity/syntaxTests/viewPureChecker/call_options_without_call.sol

```diff
@@ -0,0 +1,12 @@
+contract C {
+    function f() external payable {}
+	function g(address a) external pure {
+		a.call{value: 42};
+		a.call;
+		a.staticcall;
+		a.delegatecall;
```

Comment:
Only the first one is relevant to the test. You can remove the remaining 3 since they have no call options now.